CycloneDX/Services/NugetService.cs

```diff
@@ -99,14 +99,14 @@ public async Task<Component> GetComponentAsync(string name, string version)
 
             var nuspecFilename = GetCachedNuspecFilename(name, version);
 
-            NuspecReader nuspecReader;
+            NuspecReader nuspecReader = null;
 
             if (nuspecFilename == null)
             {
                 var url = _baseUrl + name + "/" + version + "/" + name + ".nuspec";
                 using (var xmlStream = await _httpClient.GetXmlStreamAsync(url))
                 {
-                    nuspecReader = new NuspecReader(xmlStream);
+                    if (xmlStream != null) nuspecReader = new NuspecReader(xmlStream);
```

Comment:
Does this solve #74

I stepped through the code when debugging and had a hunch it might have been this.

Reply:
Nvm I just saw you linking it to the issue :) thanks for the swift work!